common/app/views/fragments/javaScriptFirstSteps.scala.html

```diff
@@ -194,9 +194,19 @@
 
 </script>
 
-@* Not the usual type of thing we do, but I want this separate to our "normal" javascript. *@
-@* also delete iPhoneConfidenceCheck inside of facia.js *@
+
+<script>
+    guardian.isIphone6 = false;
+    guardian.isIphone4 = false;
+    guardian.inTestBucket = Math.floor(Math.random() * 2 + 1) === 1;
```

Comment:
True, lots of this code is hacky. My only defence is its just some debug code and will be deleted as soon as we get to the bottom of the crashing problem :frowning: